### PR TITLE
[_]: feat/change-currency

### PR DESCRIPTION
**Pull request by xabg2**
This PR provides a country to guess the local currency. Changes: 

- `getCountryCode` function added.
- Currency change to € or $ depends on the country.

Environment variables to add:
- `NEXT_BRIDGE_URL` -> `NEXT_STORAGE_API_URL`
- `NEXT_PUBLIC_COUNTRY_API_URL`
- `NEXT_DRIVE_WEB`
- `SEGMENT_API_KEY_PROD`

### CODE REVIEW

File: pages/pricing.js [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
+    const url = `${process.env.NEXT_PUBLIC_BRIDGE_URL}/drive/device/geolocation`;
+    let credentials = (`${process.env.NEXT_PUBLIC_AUTH_BASIC_USERNAME}:${process.env.NEXT_PUBLIC_AUTH_BASIC_PASSWORD}`);
+    const resp = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Authorization': `Basic ${window.btoa(credentials)}`,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        ip: "136.127.209.48"
```
Comment (sg-gs) [inline pages/pricing.js]: what is this hardcoded ip?

File: lib/utils.ts [hunk 2]
```diff
@@ -12,13 +12,13 @@ function getBridgeAuth(): { username: string, password: string } {
 export function getUserId(email: string) {
   const auth = getBridgeAuth();
 
-  return axios.post(`${process.env.NEXT_BRIDGE_URL}/gateway/uuid`, { email },
+  return axios.post(`${process.env.NEXT_PUBLIC_BRIDGE_URL}/gateway/uuid`, { email },
```
Comment (sg-gs) [inline lib/utils.ts]: NEXT_PUBLIC_BRIDGE_URL -> NEXT_STORAGE_API_URL

File: pages/pricing.js [hunk 3]
```diff
@@ -1,22 +1,52 @@
...
+      .then(response => response.json())
+      .then(data => {
+        return data;
+      });
+    return resp;
+  }
+
+  useEffect(() => {
+    getLocation()
+      .then(data => {
```
Comment (sg-gs) [inline pages/pricing.js]: add a `catch`

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: pages/pricing.js (hunk 1 above)
Comment (xabg2) [reply pages/pricing.js]: It's just a random ip for fetch testing, I eliminate it.

File: pages/pricing.js [hunk 4]
```diff
@@ -1,19 +1,39 @@
...
+    };
+    const countryCode = await axios(options);
+    return countryCode;
+  }
+
+  useEffect(() => {
+    getCountryCode()
+      .then((res) => {
+        setCountry(res.data.country);
+      })
```
Comment (sg-gs) [inline pages/pricing.js:28]: missing a `catch` here.

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: pages/pricing.js (hunk 4 above)
Comment (xabg2) [reply pages/pricing.js:28]: I have forgotten about it again

**Comment by sg-gs** [comment]
Pending to test it via VPN. I will return with the result once tested. Apart from that, LGTM.

**Comment by sg-gs** [comment]
Feedback: Be careful with the environment variables that change. As there is more than one environment variable appearing as new or being renamed. I have updated the PR's description with those changes.

**Review by sg-gs — APPROVED** [review]

**Pull request merged**